srsue/src/stack/upper/nas_5g_procedures.cc

```diff
@@ -65,11 +65,7 @@ srsran::proc_outcome_t nas_5g::pdu_session_establishment_procedure::init(const u
 srsran::proc_outcome_t nas_5g::pdu_session_establishment_procedure::react(
     const srsran::nas_5g::pdu_session_establishment_accept_t& pdu_session_est_accept)
 {
-  // TODO check the pdu session values
-  if (pdu_session_est_accept.dnn_present == false) {
```

Comment:
leave the if maybe but add `and not cfg.emergency_registration`?

Reply:
I added a new member to the nas_5g_procedure called emegency_pdu_session that was intialized during init function from the emergency_registration argument.